packages/core/src/App/Containers/Layout/header/traders-hub-home-button.tsx

```diff
@@ -13,15 +13,12 @@ const TradersHubHomeButton = observer(() => {
     const history = useHistory();
     const location = useLocation();
     const { pathname } = location;
-    const { is_next_wallet_enabled, is_next_tradershub_enabled } = useFeatureFlags();
+    const { is_next_tradershub_enabled } = useFeatureFlags();
 
     const redirect_routes = () => {
-        if (is_next_wallet_enabled) {
```

Comment:
should that be removed?

Reply:
As I remember I've done this because when the account is not migrated but the `next_wallet` flag is enabled we have a weird situation when the button sends us to the "You have no wallet account screen"

Reply:
When you would expect to go to wallets

Reply:
We're gonna remove the `next_wallet` flag either way though i think